python/lsst/ip/isr/isrTask.py

```diff
@@ -610,8 +621,10 @@ def saturationDetection(self, exposure, amp):
         """
         if not math.isnan(amp.getSaturation()):
             maskedImage = exposure.getMaskedImage()
-            dataView = maskedImage.Factory(maskedImage, amp.getRawBBox())
-            isr.makeThresholdMask(
+            for box in (amp.getRawBBox(), amp.getRawHorizontalOverscanBBox(),\
+                        amp.getRawVerticalOverscanBBox()):
+                dataView = maskedImage.Factory(maskedImage, box)
+                isr.makeThresholdMask(
                 maskedImage = dataView,
```

Comment:
Regarding the spaces. It seems that the LSST python style guide inexplicably insists on the spaces so you have to leave them in. It's the indenting that is the main problem.